doc/src/guides/solving/solve-matrix-equation.md

```diff
@@ -0,0 +1,119 @@
+# Solve a Matrix Equation
+
+Use SymPy to solve a matrix (linear) equation. For example, solving $
+\left[\begin{array}{cc} c & d\\1 & -e\end{array}\right] \left[\begin{array}{cc}
+x\\y\end{array}\right] = \left[\begin{array}{cc} 2\\0\end{array}\right] $ yields
+$ \left[\begin{array}{cc} x\\y\end{array}\right] = \left[\begin{array}{cc}
+\frac{2e}{ce+d}\\\frac{2}{ce+d}\end{array}\right]$.
+
+Alternatives to consider:
+- Solving a matrix equation is equivalent to solving a system of linear
+  equations, so if you prefer you can use *link to solve a system of equations
+  once published*
+- *alternative 2*
+
+Here is an example of solving a matrix equation with SymPy's {meth}`~.LUsolve`.
+We use the standard matrix equation formulation $Ax=b$ where
+- $A$ is the matrix representing the coefficients in the linear equations
+- $x$ is the column vector of unknowns to be solved for
+- $b$ is the column vector of constants, where each row is the value of an
+  equation
+
+```py
+>>> from sympy import symbols
+>>> from sympy.matrices import Matrix
+>>> c, d, e = symbols("c, d, e")
+>>> A = Matrix([[c,d], [1, -e]])
+>>> A
+Matrix([
+[c,  d],
+[1, -e]])
+>>> b = Matrix([2, 0])
+>>> b
+Matrix([
+[2],
+[0]])
+>>> A.LUsolve(b)
+Matrix([
+[2*e/(c*e + d)],
+[  2/(c*e + d)]])
+```
+
+## Guidance
+
+### *Guidance 1*
+
+*Guidance 1 content*
+
+### *Guidance 2*
+
+*Guidance 2 content*
+
+
+## *Title*
+
+You can *title* in several ways. 
+
+### *Method 1*
+
+*Method 1 content*
+
+### *Method 2*
+
+*Method 2 content*
+
+## Use the Solution Result
+
+### *Usage Method 1*
+
+*Usage method 1 content*
+
+### *Usage Method 2*
+
+*Usage method 2 content*
+
+## *Tradeoffs (speed vs. accuracy, etc.) for function*
+
+### *Tradeoff 1*
+
+*Tradeoff 1 content*
+
+### *Tradeoff 2*
+
+*Tradeoff 2 content*
+
+## Not All Equations Can Be Solved
+
+### Equations With No Solution
+
+If the determinant of a matrix is zero, matrix equations with it have no
+solution:
+
+```py
+>>> from sympy import symbols
+>>> from sympy.matrices import Matrix
+>>> c, d, e = symbols("c, d, e")
+>>> A = Matrix([[c*e**2, d*e], [c*e, d]])
+>>> A
+Matrix([
+[c*e**2, d*e],
+[   c*e,   d]])
+>>> b = Matrix([2, 0])
+>>> A.LUsolve(b)
+Traceback (most recent call last):
+    ...
+NonInvertibleMatrixError: Matrix det == 0; not invertible.
+```
+
+### Equations With No Analytical Solution
+
+*Equations with no analytical solution content*
+
+### Equations Which Have an Analytical Solution, and SymPy Cannot Solve
+
+*Equations which have an analytical solution, and SymPy cannot solve content*
```

Comment:
I can't think of an example where a matrix equation (a system of linear equations) has a solution, but not a closed-form one. Is there an example? (If not, I'll delete these two sections.)